File: app/src/main/java/com/mikhailovskii/trakttv/data/api/MovieAPI.java
package com.mikhailovskii.trakttv.data.api;

import android.support.annotation.NonNull;

import com.mikhailovskii.trakttv.data.entity.Movie;
import com.mikhailovskii.trakttv.data.entity.MovieTrack;

import java.util.List;

import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface MovieAPI {

    @GET("/movies/trending")
    @NonNull
    Observable<List<MovieTrack>> getMovies();

    @GET("/movies/{slug}?extended=full")
    @NonNull
    Observable<Movie> getExtendedInfo(@Path("slug") String slug);

}

File: app/src/main/java/com/mikhailovskii/trakttv/data/api/MovieAPI.kt
package com.mikhailovskii.trakttv.data.api

import com.mikhailovskii.trakttv.data.entity.Movie
import com.mikhailovskii.trakttv.data.entity.MovieTrack

import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Path

interface MovieAPI {

    @GET("/movies/trending")
    fun getMovies(): Observable<List<MovieTrack>>

    @GET("/movies/{slug}?extended=full")
    fun getExtendedInfo(
            @Path("slug") slug: String
    ): Observable<Movie>

}